ui/package.json

```diff
@@ -33,18 +33,18 @@
     "time-grunt": "1.1.1",
     "grunt-karma": "0.10.1",
     "karma-script-launcher": "0.1.0",
-    "karma-chrome-launcher": "0.1.8",
+    "karma-chrome-launcher": "0.2.3",
     "karma-safari-launcher": "0.1.1",
-    "karma-firefox-launcher": "0.1.4",
+    "karma-firefox-launcher": "0.1.7",
     "karma-html2js-preprocessor": "0.1.0",
-    "karma-jasmine": "0.3.5",
+    "karma-jasmine": "0.3.8",
     "karma-coffee-preprocessor": "0.2.1",
     "karma-requirejs": "0.2.2",
     "karma-phantomjs-launcher": "0.1.4",
     "karma": "0.12.31",
     "karma-ng-html2js-preprocessor": "0.1.2",
     "phantomjs": "1.9.16",
-    "protractor": "2.0.0",
+    "protractor": "3.2.2",
```

Comment:
Maybe we should keep the protractor at 2.x.x for now... The Chrome driver new protractor downloads after unzipping etc. has the name `chromedriver_2.21` while protractor expects it to be `chromedriver` hence e2e tests cannot be run.
If `protractor` version becomes `3.x.x` it's possible we should bump up `grunt-protractor-runner` to `3.x.x`  too.